stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/call/RtcSession.kt

```diff
@@ -223,23 +222,29 @@ public class RtcSession internal constructor(
             null,
         )
 
-    val trackDimensions =
-        MutableStateFlow<Map<String, Map<TrackType, TrackDimensions>>>(
-            emptyMap(),
-        )
-    val trackDimensionsDebounced = trackDimensions.debounce(100)
-
     // run all calls on a supervisor job so we can easily cancel them
     private val supervisorJob = SupervisorJob()
     private val coroutineScope = CoroutineScope(clientImpl.scope.coroutineContext + supervisorJob)
 
+    internal val defaultVideoDimension = VideoDimension(1080, 2340)
```

Comment:
why not full HD?

Reply:
That was the value, I just moved the property, didn't touch the actual value.